src/main/java/de/teamlapen/vampirism/entity/vampire/EntityVampireBase.java

```diff
@@ -145,7 +145,7 @@ public boolean isGettingSundamage(boolean forceRefresh) {
 
     @Override
     public boolean isIgnoringSundamage() {
-        return this.isPotionActive(ModPotions.sunscreen);
+        return this.isPotionActive(ModPotions.sunscreen) || this.ignoreSundamageAction;
```

Comment:
Is it possible to just use the potion for the entity action instead of having an additional attribute here?